packages/common/http/src/request.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import { XhrFactory } from '@angular/common/public_api';
```

Comment:
Never import from `public_api`. 
```suggestion
import { XhrFactory } from '@angular/common';
```